### PR TITLE
Fix footer link and make them scroll to top

### PR DESCRIPTION
**Pull request by smyzin**
- fixed footer links
- created Scroll component

### CODE REVIEW

File: src/components/Text.tsx [hunk 1]
```diff
@@ -266,24 +276,33 @@ export function LinkText({
...
-  fontWeight('font-semibold'),
-  textDecoration('no-underline', 'hover:underline'),
-  textColor('text-formal-accent', 'hover:text-accent')
-)
-export function FooterlLink({
+const footerLink = (active?: boolean) =>
+  classnames(
+    fontSize('text-sm'),
+    fontWeight('font-semibold'),
+    textDecoration(active ? 'underline' : 'no-underline', 'hover:underline'),
```
Comment (T-Damer) [inline src/components/Text.tsx:283]: Let's do object here

File: src/components/Text.tsx [hunk 2]
```diff
@@ -266,24 +276,33 @@ export function LinkText({
...
+    textColor(
+      active ? 'text-accent' : 'text-formal-accent',
```
Comment (T-Damer) [inline src/components/Text.tsx:285]: Let's do object here

**Review by T-Damer — APPROVED** [review]

File: src/components/Text.tsx (hunk 1 above)
Comment (smyzin) [reply src/components/Text.tsx]: Fixed

File: src/components/Text.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/Text.tsx]: Fixed

**Pull request merged**